substra/cli/interface.py

```diff
@@ -27,20 +27,20 @@
 from substra.sdk import user as usr
 
 
-def get_client(config_path, profile_name, user_path):
+def get_client(params):
```

Comment:
Why call it params here instead of global_conf as you did everywhere else?

Reply:
Yes, I will rename it! :)